src/pybind/mgr/prometheus/module.py

```diff
@@ -241,6 +247,24 @@ def _setup_static_metrics(self):
             'MON Metadata',
             MON_METADATA
         )
+        metrics['mgr_metadata'] = Metric(
+            'gauge',
+            'mgr_metadata',
+            'MGR metadata',
+            MGR_METADATA
+        )
+        metrics['mgr_status'] = Metric(
+            'gauge',
+            'mgr_status',
+            'MGR status (0=standby, 1=active)',
+            MGR_STATUS
+        )
+        metrics['mgr_module_status'] = Metric(
+            'gauge',
+            'mgr_module_status',
+            'MGR module status (0=disabled, 1=enabled)',
+            MGR_MODULE_STATUS
```

Comment:
I think it would make sense to separate the `can_run` label into its own metric. This is of interest to admins, rather then just metadata.

Reply:
Could you explain why something like ceph_mgr_module_status{can_run='1'} == 0 would not be sufficient?

Reply:
Well its certainly sufficient for most cases, it just doesn't adhere to  [prometheus' best practice](https://prometheus.io/docs/instrumenting/writing_exporters/#labels).

One things that isn't easily possible is to get a range_vector for can_run, as it is a metric dimension instead of its own metric. This might be of interest when analysing a the root cause of `can_run=0`.

Reply:
OK - if that's the use case your thinking of, I'm happy to make the changes. Will push updates in the next day or so.

Reply:
@jan--f can you please review the latest commit. I've added runnable as a separate metric and also changed status so it shows 0,1 or 2 corresponding to disabled, enabled and auto-enabled. If your happy with this I'll squash the commits in this PR.